scripts/autotools/libmongoc/CheckSSL.m4

```diff
@@ -1,23 +1,31 @@
 PHP_ARG_WITH([mongodb-ssl],
              [whether to enable crypto and TLS],
-             [AS_HELP_STRING([--with-mongodb-ssl=@<:@auto/no/openssl/libressl/darwin@:>@],
+             [AS_HELP_STRING([--with-mongodb-ssl=@<:@auto/openssl/libressl/darwin/no@:>@],
                              [MongoDB: Enable TLS connections and SCRAM-SHA-1 authentication [default=auto]])],
              [auto],
              [no])
 
+if test "$PHP_MONGODB_SSL" = "auto" -o "$PHP_MONGODB_SSL" = "no"; then
+  crypto_required="no"
+else
+  crypto_required="yes"
```

Comment:
`crypto_required` technically isn't necessary when explicitly selecting a TLS library but this made the logic for readable IMO. It _is_ relevant when we convert "yes" to "auto" since "auto" would otherwise have _not_ required a library.